_src/static/sass/components/_headings.scss

```diff
@@ -0,0 +1,128 @@
+// ------------------------------
+// edX Pattern Library: Components - Headings
+
+// About: Contains base styling for headings, relying on Bourbon's mathematical scales for precision.
+
+// Scale:
+// 12, 14, 16, 18, 21, 24, 32, 43
+// ----------------------------
+
+// #GLOBALS
+// #HEADINGS
+// H1
+// H2
+// H3
+// H4
+// H5
+// H6
+// ----------------------------
+
+
+// ----------------------------
+// #GLOBALS
+// ----------------------------
+.hd-1,
+.hd-2,
+.hd-3,
+.hd-4,
+.hd-5,
+.hd-6 {
```

Comment:
I've moved this into the reset extend. In fact, the extend had most of this so it was just removed.